src/AutoRest.CSharp.V3/Output/Models/Client.cs

```diff
@@ -30,6 +30,7 @@ public Client(OperationGroup operationGroup, BuildContext context): base(context
             var clientName = clientPrefix + ClientSuffix;
 
             DefaultName = clientName;
+            DefaultAccessibility = operationGroup.Extensions?.Accessibility ?? "public";
```

Comment:
It would be nice - though not a big deal - if we could move these as well. While we won't expose them in Search directly anyway, I can imagine some clients might expose the generated ones and may not need/want this.

Reply:
Leaving this was a mistake. Unfortunately there is no single place in swagger that represents a "client" so there is nothing to apply namespace to, I'll revert this file and keep the feature about model for now.

Reply:
Ah, yes, that's a good point.